src/testing.rs

```diff
@@ -290,3 +290,7 @@ macro_rules! rb_init_items {
 pub fn lazy_eval(code: &str) -> VALUE {
     new!(*LAZY_VALUE, code.to_ruby())
 }
+
+pub fn c_bool(val: c_int) -> bool {
+    val > 0
```

Comment:
I think `!= 0` is more correct right? I _think_ -1 is treated as true

Reply:
You are correct.